src/plugin.js

```diff
@@ -309,9 +324,17 @@ const plugin = function(options) {
   // because it doesn't make sense to pass it to each Overlay component.
   delete settings.overlays;
 
-  this.overlays_ = overlays.map(
-    o => this.addChild('overlay', videojs.mergeOptions(settings, o))
-  );
+  this.overlays_ = overlays.map(o => {
+    let mergeOptions = videojs.mergeOptions(settings, o);
+
+    // Attach bottom aligned overlays to the control bar so
+    // they will adjust positioning when the control bar minimizes
+    if (this.controlBar && mergeOptions.align.indexOf('bottom') !== -1) {
```

Comment:
This would make this a breaking change. I'm not sure it's worth it.
Maybe it's better to just do it via CSS? when `vjs-user-active` is set, add a margin-bottom of `3em` or something?